### PR TITLE
Implementation of feedback

### PR DESCRIPTION
**Pull request by EmefileFrancis**
@torkpe Please help me review before I merge.
The following was implemented as required by your feedback:

1. Transpile the test script with Babel before running the test, so that I could use ES6 **import** and **export** in **test/entriesTest.js**

2. Remove the for-loop in the test script, and test for the first and last items in the entries array in  **test/entriesTest.js**

3. After deleting, return all entries in **src/routes/entries.js**

4. After modifying, return all entries  in **src/routes/entries.js**

5. Always return objects as the response body when there is an error in **src/routes/entries.js**

### CODE REVIEW

File: dist/src/routes/entries.js [hunk 1]
```diff
@@ -22,15 +22,15 @@ router.get('/:id', function (req, res) {
   var entry = _entry.entries.find(function (c) {
     return c.id === parseInt(req.params.id, 10);
   });
-  if (!entry) res.status(404).send('Entry with specified ID not found.');
+  if (!entry) res.status(404).send({ message: 'Entry with specified ID not found.' });
```
Comment (EmefileFrancis) [inline dist/src/routes/entries.js]: I made a change here. Sent an object instead of a string.

File: dist/src/routes/entries.js [hunk 2]
```diff
@@ -22,15 +22,15 @@ router.get('/:id', function (req, res) {
...
   res.send(entry);
 });
 
 router.get('/userId/:userId', function (req, res) {
   var entriesByUser = _entry.entries.filter(function (c) {
     return c.userId === parseInt(req.params.userId, 10);
   });
-  if (entriesByUser === undefined || entriesByUser.length === 0) res.sendStatus(404).send('No entry with specified userId.');
+  if (entriesByUser === undefined || entriesByUser.length === 0) res.status(404).send({ message: 'Entry with specified ID not found.' });
```
Comment (EmefileFrancis) [inline dist/src/routes/entries.js]: I made a change here. Sent an object instead of a string.

File: dist/src/routes/entries.js [hunk 3]
```diff
@@ -53,7 +53,7 @@ router.put('/:id', function (req, res) {
   var entry = _entry.entries.find(function (c) {
     return c.id === parseInt(req.params.id, 10);
   });
-  if (!entry) res.status(404).send('Entry with specified Id not found.');
+  if (!entry) res.status(404).send({ message: 'Entry with specified ID not found.' });
```
Comment (EmefileFrancis) [inline dist/src/routes/entries.js]: I made a change here. Sent an object instead of a string.

File: dist/src/routes/entries.js [hunk 4]
```diff
@@ -63,18 +63,18 @@ router.put('/:id', function (req, res) {
...
-  res.send(entry);
+  res.send(_entry.entries);
 });
 
 router.delete('/:id', function (req, res) {
   var entry = _entry.entries.find(function (c) {
     return c.id === parseInt(req.params.id, 10);
   });
-  if (!entry) res.status(404).send('Entry with specified Id not found.');
+  if (!entry) res.status(404).send({ message: 'Entry with specified ID not found.' });
```
Comment (EmefileFrancis) [inline dist/src/routes/entries.js]: I made a change here. Sent an object instead of a string.

File: dist/src/routes/entries.js [hunk 5]
```diff
@@ -63,18 +63,18 @@ router.put('/:id', function (req, res) {
   entry.body = req.body.body;
   entry.date = req.body.date;
 
-  res.send(entry);
+  res.send(_entry.entries);
```
Comment (EmefileFrancis) [inline dist/src/routes/entries.js]: Sent the entries array instead of entry

**Review by torkpe — CHANGES_REQUESTED** [review]
You do not need to push your nyc folder

File: .nyc_output/17275a6c299880e17d135b4cd9dcb28a.json [hunk 6]
```diff
@@ -0,0 +1 @@
...":21,"column":2},"end":{"line":21,"column":56}}},"fnMap":{"0":{"name":"(anonymous_0)","decl":{"start":{"line":7,"column":18},"end":{"line":7,"column":19}},"loc":{"start":{"line":7,"column":33},"end":{"line":10,"column":1}},"line":7},"1":{"name":"_interopRequireDefault","decl":{"start":{"line":20,"column":9},"end":{"line":20,"column":31}},"loc":{"start":{"line":20,"column":37},"end":{"line":22,"column":1}},"line":20}},"branchMap":{"0":{"loc":{"start":{"line":21,"column":9},"end":{"line":21,"column":55}},"type":"cond-expr","locations":[{"start":{"line":21,"column":33},"end":{"line":21,"column":36}},{"start":{"line":21,"column":39},"end":{"line":21,"column":55}}],"line":21},"1":{"loc":{"start":{"line":21,"column":9},"end":{"line":21,"column":30}},"type":"binary-expr","locations":[{"start":{"line":21,"column":9},"end":{"line":21,"column":12}},{"start":{"line":21,"column":16},"end":{"line":21,"column":30}}],"line":21}},"s":{"0":1,"1":1,"2":1,"3":1,"4":1,"5":1,"6":1,"7":1,"8":2},"f":{"0":1,"1":2},"b":{"0":[0,2],"1":[2,2]},"_coverageSchema":"d34fc3e6b8297bcde183f5492bcb8fcb36775295","hash":"803353c7e4382410221fca439e66bae824d30b76","contentHash":"798e40db6a8ff494bebb30c327ad4122_12.0.2"}}
```
Comment (torkpe) [inline .nyc_output/17275a6c299880e17d135b4cd9dcb28a.json]: you don't need to push your nyc reports

File: test/entriesTest.js [hunk 7]
```diff
@@ -70,12 +65,12 @@ describe('api/v1/entries', () => {
       expect(entries.length).to.equal(2);
     });
 
-    it('should return the deleted entry', async () => {
-      const res = await request(server.default).delete('/api/v1/entries/2');
+    // it('should return the deleted entry', async () => {
```
Comment (torkpe) [inline test/entriesTest.js]: Why are you still pushing commented code blocks?

File: coverage/dist/src/routes/entries.js.html [hunk 8]
```diff
@@ -0,0 +1,309 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <title>Code coverage report for dist/src/routes/entries.js</title>
```
Comment (torkpe) [inline coverage/dist/src/routes/entries.js.html]: Please do well to indent your code properly

**Review by torkpe — CHANGES_REQUESTED** [review]

**Review by torkpe — APPROVED** [review]
LGTM

**Pull request merged**